documentation/PYTHON.rst

```diff
@@ -1,11 +1,11 @@
 ****************
-Python Interface
+Python interface
```

Comment:
Why remove capitalization here (and below)?

Reply:
Because it was inconsistently used. (And I prefer only initial uppercase ;))

Can life with either, if consistently.

Reply:
okay then lets keep it like that for now.